.travis.yml

```diff
@@ -2,12 +2,13 @@ language: node_js
 node_js:
   - "node" # Latest node version
   - "lts/*" # Latest LTS version
+  - "12"
   - "10"
   - "8"
-  - "8.10"
-  - "6"
 
-before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.4.1; fi
+before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.13.4; fi
```

Comment:
I'm thinking we are going to want to remove this `if` statement here, and from any open source repo moving forward.